merlin/systems/dag/ops/faiss.py

```diff
@@ -59,7 +59,7 @@ def __init__(self, index_path, topk=10):
         super().__init__()
 
     @classmethod
-    def from_config(cls, config: dict) -> "QueryFaiss":
+    def from_config(cls, config: dict, **kwargs) -> "QueryFaiss":
```

Comment:
Whats the benefit of adding the `kwargs` here ?

Reply:
This is required with the change made in `op_runner.py`. The from_config now receives `model_repository`, `model_name`, and `model_version` keyword arguments which enables us to read files stored alongside the model config.

This QueryFaiss operator in this case relies on the config containing the full path to the index file currently. However with this extra information could be updated to read from a location relative to the model repository and model config.

It could be clearer to update this **kwargs to specify the named arguments explicitly here (and in the other from_config methods.)